bib/cmd/bootc-image-builder/main.go

```diff
@@ -125,47 +112,79 @@ func getContainerSize(imgref string) (uint64, error) {
 	return size, nil
 }
 
-func makeManifest(c *ManifestConfig, cacheRoot string) (manifest.OSBuildManifest, error) {
-	if c.Local {
-		if err := setup.ValidateHasContainerStorageMounted(); err != nil {
-			return nil, fmt.Errorf("local storage not working, did you forget -v /var/lib/containers/storage:/var/lib/containers/storage? (%w)", err)
+// removeDuplicateRepos removes duplicities from the list of repositories
+//
+// The equality of repositories is determined by the unique Id field.
+func removeDuplicateRepos(repos []rpmmd.RepoConfig) []rpmmd.RepoConfig {
+	ids := make(map[string]bool)
+	dedupedRepos := []rpmmd.RepoConfig{}
+	for _, repo := range repos {
+		if _, value := ids[repo.Id]; !value {
+			ids[repo.Id] = true
+			dedupedRepos = append(dedupedRepos, repo)
 		}
 	}
+	return dedupedRepos
+}
 
+type mTLSConfig struct {
+	keyPath  string
+	certPath string
+	caPath   string
+}
+
+func makeManifest(c *ManifestConfig, cacheRoot string) (manifest.OSBuildManifest, *mTLSConfig, error) {
 	// If --local wasn't given, always pull the container.
```

Comment:
This comment no longer applies, does it? we remove the `c.Local` checks here (and move them down)?

Reply:
Removed, thanks! :)